homeassistant/components/netgear_lte/entity.py

```diff
@@ -1,54 +1,33 @@
 """Entity representing a Netgear LTE entity."""
-
-from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import CONF_HOST
 from homeassistant.helpers.device_registry import DeviceInfo
-from homeassistant.helpers.dispatcher import async_dispatcher_connect
-from homeassistant.helpers.entity import Entity, EntityDescription
+from homeassistant.helpers.entity import EntityDescription
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
 
-from . import ModemData
-from .const import DISPATCHER_NETGEAR_LTE, DOMAIN, MANUFACTURER
+from .const import DOMAIN, MANUFACTURER
+from .coordinator import NetgearLTEDataUpdateCoordinator
 
 
-class LTEEntity(Entity):
+class LTEEntity(CoordinatorEntity[NetgearLTEDataUpdateCoordinator]):
     """Base LTE entity."""
 
     _attr_has_entity_name = True
-    _attr_should_poll = False
 
     def __init__(
         self,
-        config_entry: ConfigEntry,
-        modem_data: ModemData,
+        coordinator: NetgearLTEDataUpdateCoordinator,
         description: EntityDescription,
     ) -> None:
         """Initialize a Netgear LTE entity."""
+        super().__init__(coordinator)
         self.entity_description = description
-        self.modem_data = modem_data
-        self._attr_unique_id = f"{description.key}_{modem_data.data.serial_number}"
+        self._attr_unique_id = f"{description.key}_{coordinator.data.serial_number}"
```

Comment:
You're using coordinator.data quite a lot, maybe you can use a local variable to avoid accessing the variable a lot

Reply:
Just for aesthetics? It doesn't save any lines by doing that.

Reply:
It doesn't save lines but it looks better and is a bit more performant